portfolio/pom.xml

```diff
@@ -16,7 +16,7 @@
 
     <!-- Project-specific properties -->
     <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
-    <googleCloudProjectId>YOUR_PROJECT_ID_HERE</googleCloudProjectId>
+    <googleCloudProjectId>alowfung-sps-spring21</googleCloudProjectId>
```

Comment:
The server side functionality is not part of the frontend and should be a different PR. Remember to make a new PR from the *main* branch. Make a new PR for each feature. Subsequent commits after the first one to the same PR should only be to resolve review comments. New feature should mean new PR.